src/interop/passes.jl

```diff
@@ -34,3 +34,9 @@ demote_float16!(pm::PassManager) = API.LLVMAddDemoteFloat16Pass(pm)
 remove_ni!(pm::PassManager) = API.LLVMAddRemoveNIPass(pm)
 
 julia_licm!(pm::PassManager) = API.LLVMAddJuliaLICMPass(pm)
+
+if VERSION >= v"1.8.0-DEV.1082"
+    cpu_features!(pm::PassManager) = API.LLVMAddCPUFeaturesPass(pm)
+else
+    cpu_features!(pm::PassManager) = (@warn("CPU features pass not available"); nothing)
```

Comment:
Is that consistent with how we handle other passes? They should probably warn too.

Reply:
Apparently we don't handle it for the other passes xD 
which might be because we are not supporting old enough Julia's?

I think we used to just return `nothing`.